src/game_frontend/src/components/ui/Sidebar/index.css

```diff
@@ -1,5 +1,5 @@
 .sidebar{
-    @apply fixed w-auto min-h-screen h-auto left-0 top-0 flex flex-col justify-center;
+    @apply fixed w-auto min-h-screen h-auto left-[-12px] top-0 flex flex-col justify-center;
```

Comment:
Why was `left-[-12px]` used?

Reply:
To just look like figma i used it

Reply:
![{48AE42A3-F05C-4B26-98A0-2237D4E08BD9}](https://github.com/user-attachments/assets/913f13ce-a358-4290-becb-38594e78df01)
no gap was there so i thought to give -12px